### PR TITLE
Fix repository name not populated

### PR DESCRIPTION
**Pull request by edaena**
Repository name is not part of the supported "opts"

### CODE REVIEW

**Review by samiyaakhtar — APPROVED** [review]

File: src/commands/project/pipeline.ts [hunk 1]
```diff
@@ -143,7 +143,7 @@ export const fetchValidateValues = (
     orgName: opts.orgName || "",
```
Comment (mtarng) [inline src/commands/project/pipeline.ts]: This whole logic section is kinda confusing. Why are we still loading from `opts` from here. Shouldn't all of these be loaded from the `values` object instead? 

This was introduced in: https://github.com/CatalystCode/spk/pull/479/files

@edaena @dennisseah does  this make sense for us to fix this all here? Does similar logic for other ConfigValues interfaces exist now too in other commands?

**Review by mtarng — CHANGES_REQUESTED** [review]

File: src/commands/project/pipeline.ts (hunk 1 above)
Comment (dennisseah) [reply src/commands/project/pipeline.ts]: @mtarng do you know other places that this problem too?

**Review by dennisseah — APPROVED** [review]

File: src/commands/project/pipeline.ts (hunk 1 above)
Comment (mtarng) [reply src/commands/project/pipeline.ts]: I'm not sure, which is why I asked the question @dennisseah . Have we refactored any other commands to move arguments from `CommandOptions` to `ConfigValues`?

**Review by mtarng — APPROVED** [review]

File: src/commands/project/pipeline.ts (hunk 1 above)
Comment (dennisseah) [reply src/commands/project/pipeline.ts]: I will double check

**Pull request merged**